src/daemon/engine/svc/node/list_registers_service.hpp

```diff
@@ -0,0 +1,38 @@
+//
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+//
+
+#ifndef OCVSMD_DAEMON_ENGINE_SVC_NODE_LIST_REGISTERS_SERVICE_HPP_INCLUDED
+#define OCVSMD_DAEMON_ENGINE_SVC_NODE_LIST_REGISTERS_SERVICE_HPP_INCLUDED
+
+#include "svc/svc_helpers.hpp"
+
+namespace ocvsmd
+{
+namespace daemon
+{
+namespace engine
+{
+namespace svc
+{
+namespace node
+{
+
+/// Defines registration factory of the 'Node: List Registers' service.
```

Comment:
What's the "factory" part of this?

Reply:
Hmm..., `Impl{context}` construction at line @ 337 ?